terminology-guides/german-terminology.md

```diff
@@ -39,34 +39,43 @@ This assumes that numeric value input defaults to US throughout localizations.
 
 | English | German | Notes |
 | --- | --- | --- |
-| local daemon startup flags | | |
-| signing address | | recherchieren |
-| Spendable Wallet | | recherchieren |
-| Tx descriptions | | |
-| wallet creation height | | |
+
 
 ## New
 
 | English | German | Notes |
 | --- | --- | --- |
-| **cache** | Cache | |
-| **daemon log path** | Daemon-Log-Pfad | |
-| **hidden** | versteckt | |
-| **QrCode(Qr Code)** ^1 | QR-Code | |
-| **spent outputs** | verbrauchte Outputs | |
-| **view key** | View-Key | really new? |
-| **wallet log path** | Wallet-Log-Pfad | |
-
-**^1** : Inconsistent in upstream?
+| | | |
+| **Not in GUI** | | |
+| ring confidential transaction | vertrauliche Ringtransaktion | |
+| (Ring CT) | Ring-CT | |
+| block emission | Verteilung(skurve) | |
+| main (block) emission | Hauptverteilung | |
+| tail (block) emission | Anschlussverteilung | |
+| testnet | Testnet | |
+| transaction unlock time | Freigabedauer von Transaktionen | |
 
 ## Discuss Change
 
 | English | German | Notes |
 | --- | --- | --- |
-| clear | | is: _leeren_ |
-| Generate | _erzeugen_ | was: _generieren/erstellen_ |
-| money | | is: _Geld/Guthaben_ |
+| clear | zurücksetzen | is: _leeren_ |
+| daemon log path | Hintergrunddienst-Log-Pfad | |
+| money | Geld | is: _Geld/Guthaben_ |
+| node | der/das Node | please discuss which article should be used; use _der Node_ for now |
+| local daemon startup flags | Startparameter für lokalen Hintergrunddienst |  |
+| signing address | signierende Adresse | |
+| Spendable Wallet | Wallet mit Vollzugriff | |
+| spent outputs | ausgegebene Outputs | |
 | Tracking | | is: _Nachverfolgen_ |
+| Tx descriptions | Transaktionsbeschreibungen | |
+| wallet | die/das Wallet | please discuss which article should be used; use _das Wallet_ for now |

```

Comment:
I don't know if yo uare still interested in pushing this PR, but if you are, please change the second column to "das wallet".